src/pages/pricing/index.tsx

```diff
@@ -43,92 +44,93 @@ const Pricing: FunctionComponent<PricingProps> & {getLayout: any} = () => {
     send,
     priceId,
     quantity,
-    prices,
+    // prices,
     availableCoupons,
-    currentPlan,
+    // currentPlan,
   } = useCommerceMachine()
```

Comment:
Since this is completely contained in `PricingWidget` now, it can be deleted from here.